docs/whatsnew/2.9.rst

```diff
@@ -1,6 +1,44 @@
 What's new in h5py 2.9
 ======================
 
+New features
+------------
+
+* A convenient high-level API for creating virtual datasets, HDF5 objects which
+  act as a view over one or more real datasets (:issue:`1060`, :issue:`1126`).
+  See :ref:`vds` for details.
+* :class:`File` can now be constructed with a Python file-like object,
+  making it easy to create an HDF5 file in memory using :class:`io.BytesIO`
+  (:issue:`1061`, :issue:`1105`, :issue:`1116`).
+  See :ref:`file_fileobj` for details.
+* :class:`File` now accepts parameters to control the chunk cache
+  (:issue:`1008`). See :ref:`file_cache` for details.
+* New options to record the order of insertion for attributes and group entries.
+  Iterating over these collections now follows insertion order if it was
+  recorded, or name order if not (:issue:`1098`).
+* A new method :meth:`Group.create_dataset_like` to create a new dataset with
+  similar properties to an existing one (:issue:`1085`).
+* Datasets can now be created with storage backed by external non-HDF5 files
+  (:issue:`1000`).
+* Lists or tuples of unicode strings can now be stored as HDF5 attributes
+  (:issue:`1032`).
+* Inspecting the view returned by ``.keys()`` now shows the key names, for
+  convenient interactive use (:issue:`1049`).
+
+Exposing HDF5 functions
+-----------------------
+
+* ``H5LTopen_file_image`` as :func:`h5py.h5f.open_file_image` (:issue:`1075`)
+* External dataset storage functions ``H5Pset_external``, ``H5Pget_external``
+  and ``H5Pget_external_count`` as methods on :class:`h5py.h5p.PropDCID`
+  (:issue:`1000`)
+
+Bugfixes
+--------
+
+* Reading float128 data on ppc64 architecture systems (:issue:`1114`)
```

Comment:
@tacaswell is this correct? You mentioned Windows in the TODO list, but I didn't see anything about that in the PR, and the changes looked like they would only affect ppc64.

Reply:
The OP reports is is a problem on linux as well (and from my comments I could apparently reproduce it locally on x86_64 linux).  I think that what happened was that the fixes for ppc64 elsewhere ended up breaking float128 in other ways.  I think #1114 gets it to work correctly everywhere we need and adds round-trip tests for the platforms we test on.

Hopefully the RC will be built by the debian folks who have some more exotic platforms.